internal/configprovider/manager.go

```diff
@@ -514,12 +514,12 @@ func parseCfgSrc(s string) (cfgSrcName, selector string, params interface{}, err
 		selector = strings.Trim(parts[0], " ")
 
 		if len(parts) > 1 && len(parts[1]) > 0 {
-			v := config.NewParser().Viper()
-			v.SetConfigType("yaml")
-			if err = v.ReadConfig(bytes.NewReader([]byte(parts[1]))); err != nil {
+			var v *config.Parser
```

Comment:
Same change coming via https://github.com/signalfx/splunk-otel-collector/pull/362/files#diff-0b8dacdc445e88bb05f90c35b7529cb373e6f3af1b44949834e1d4e5b6ea5812R538 
```suggestion
			var p *config.Parser
```
to avoid conflicts

Reply:
Done.